.github/workflows/ci_tests.yml

```diff
@@ -1,49 +0,0 @@
-name: CI tests
-
-on: [push]
-
-jobs:
-  build:
-
-    runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 1
-      matrix:
-        python-version: [3.7]
-
-    steps:
-    - uses: actions/checkout@v1
-    - name: Get Cmake version
-      run: |
-        cmake --version
-    - name: Update submodules
-      run: |
-        git submodule init
-        git submodule update
-    - name: Install dependencies
-      run: |
-        sudo apt-get install libjansson-dev
-        sudo apt-get install libhdf5-dev
-        sudo apt-get install python3-dev
-        sudo apt-get install python3-numpy
-        sudo apt-get install python3-yaml
-        sudo apt-get install python3-psycopg2
-        sudo apt-get install swig3.0
-        sudo apt-get install doxygen
-        sudo apt-get install graphviz
-    - name: configure
-      run: |
-        mkdir build
-        cd build
-        cmake ..
-        cd -
-    - name: make
-      run: |
-        cd build
-        make
-        cd -
-    - name: make test
-      run: |
-        cd build
-        make test
```

Comment:
What is the reason for removing this workflow? Isn't is a very useful complement to the static code analyser to build and run the test suite for each push?

Reply:
Yes, but it is identical to the workflow in ci.yml, which already does testing and all that jazz. So, I thought I'd remove one of them assuming that having them both saves no real purpose 🥦 !